### PR TITLE
fix: update vulnerable dependencies

### PR DESCRIPTION
**Pull request by Sergio-svb**
- update vulnerable dependencies
- update readme

### CODE REVIEW

**Review by joseandrespg — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
 # NOTE: WIP FORK
 
-This is a fork of the `handlebars-helpers` npm created by [Jon Schlinkert](https://github.com/jonschlinkert).
+This is a fork of the [handlebars-kit](https://github.com/helpers/handlebars-helpers) created by [Jon Schlinkert](https://github.com/jonschlinkert).
```
Comment (thekiwi) [inline README.md]:
```suggestion
This is a fork of the [handlebars-helpers](https://github.com/helpers/handlebars-helpers) created by [Jon Schlinkert](https://github.com/jonschlinkert).
```

It's a fork of `handlebars-helpers` not `handlebars-kit`

File: README.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
 # NOTE: WIP FORK
```
Comment (thekiwi) [inline README.md]:
```suggestion
# Handlebars Kit

More than 150 helpers in ~15 categories designed to extend the Node.js Handlebars library.
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
 
-This is a fork of the `handlebars-helpers` npm created by [Jon Schlinkert](https://github.com/jonschlinkert).
+This is a fork of the [handlebars-kit](https://github.com/helpers/handlebars-helpers) created by [Jon Schlinkert](https://github.com/jonschlinkert).
 
-Below is the original readme. More info to come.
+- The fork minimises dependencies to reduce the maintenance effort
+- The fork removes risky helpers, such as any that interact with the file system
+- The fork is published to NPM https://www.npmjs.com/package/handlebars-kit
+- The fork has higher test coverage
```
Comment (thekiwi) [inline README.md]:
```suggestion
Motivations for forking
- To minimise the number of dependencies, reducing the maintenance effort
- To remove potentially risky helpers (such as any that interact with the file system)
- To increase the number of test cases and resulting test coverage
```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
 
----
-
-
-# handlebars-helpers [![NPM version](https://img.shields.io/npm/v/handlebars-helpers.svg?style=flat)](https://www.npmjs.com/package/handlebars-helpers) [![NPM monthly downloads](https://img.shields.io/npm/dm/handlebars-helpers.svg?style=flat)](https://npmjs.org/package/handlebars-helpers)  [![NPM total downloads](https://img.shields.io/npm/dt/handlebars-helpers.svg?style=flat)](https://npmjs.org/package/handlebars-helpers) [![Linux Build Status](https://img.shields.io/travis/helpers/handlebars-helpers.svg?style=flat&label=Travis)](https://travis-ci.org/helpers/handlebars-helpers) [![Windows Build Status](https://img.shields.io/appveyor/ci/helpers/handlebars-helpers.svg?style=flat&label=AppVeyor)](https://ci.appveyor.com/project/helpers/handlebars-helpers)
 
 > More than 130 Handlebars helpers in ~20 categories. Helpers can be used with Assemble, Generate, Verb, Ghost, gulp-handlebars, grunt-handlebars, consolidate, or any node.js/Handlebars project.
```
Comment (thekiwi) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -26,27 +25,23 @@ You might also be interested in [template-helpers](https://github.com/jonschlink
...
-
 ## Usage
 
 The main export returns a function that needs to be called to expose the object of helpers.
 
 **Get all helpers**
 
 ```js
-var helpers = require('handlebars-helpers')();
+var helpers = require('handlebars-kit')();
```
Comment (thekiwi) [inline README.md]:
```suggestion
const helpers = require('handlebars-kit')();
```

File: README.md [hunk 6]
```diff
@@ -55,11 +50,11 @@ var helpers = require('handlebars-helpers')();
 Helper collections are exposed as getters, so only the helpers you want will be required and loaded.
 
 ```js
-var helpers = require('handlebars-helpers');
+var helpers = require('handlebars-kit');
 var math = helpers.math();
```
Comment (thekiwi) [inline README.md]:
```suggestion
const helpers = require('handlebars-kit');
const math = helpers.math();
```

File: README.md [hunk 7]
```diff
@@ -55,11 +50,11 @@ var helpers = require('handlebars-helpers')();
...
 //=> only the `math` helpers
 
-var helpers = require('handlebars-helpers');
+var helpers = require('handlebars-kit');
 var array = helpers.array();
```
Comment (thekiwi) [inline README.md]:
```suggestion
const helpers = require('handlebars-kit');
const array = helpers.array();
```

File: README.md [hunk 8]
```diff
@@ -69,15 +64,15 @@ var array = helpers.array();
 Helper collections are exposed as getters, so only the helpers you want will be required and loaded.
 
 ```js
-var helpers = require('handlebars-helpers')(['math', 'string']);
+var helpers = require('handlebars-kit')(['math', 'string']);
```
Comment (thekiwi) [inline README.md]:
```suggestion
const helpers = require('handlebars-kit')(['math', 'string']);
```

File: README.md [hunk 9]
```diff
@@ -69,15 +64,15 @@ var array = helpers.array();
...
 //=> only the `math` and `string` helpers
 ```
 
 **Optionally pass your own handlebars**
 
 ```js
 var handlebars = require('handlebars');
-var helpers = require('handlebars-helpers')({
+var helpers = require('handlebars-kit')({
```
Comment (thekiwi) [inline README.md]:
```suggestion
const helpers = require('handlebars-kit')({
```

File: README.md [hunk 10]
```diff
@@ -118,7 +111,7 @@ Currently **189 helpers** in **20 categories**:
 
 ### [array helpers](#array)
 
-Visit the: [code](lib/array.js) | [unit tests](test/array.js) | [issues](https://github.com/helpers/handlebars-helpers/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+array+helpers))
+Visit the: [code](lib/array.js) | [unit tests](test/array.js)
 
 * **[after](#after)** ([code](lib/array.js#L22) | [tests](test/array.js#L13))
```
Comment (thekiwi) [inline README.md]: Are these links to the code line numbers (ie, `lib/array.js#13`) still correct, or have they changed since the fork/refactoring?

**Review by thekiwi — CHANGES_REQUESTED** [review]

**Review by thekiwi — APPROVED** [review]

**Pull request merged**